app/src/components/DraftResourceButtons.tsx

```diff
@@ -2,11 +2,15 @@ import { Text, Paper, Stack, Center, Loader } from '@mantine/core';
 import { trpc } from '@/util/trpc';
 import SidebarButtonList from './SidebarButtonList';
 
+interface MyComponentProps {
```

Comment:
I would name this to something more descriptive, maybe like `DraftResourceButtonsProps`? (@sarahmcdougall do we need this interface? Or should we just pass `review: boolean` into the function directly?